docs/UserGuide.md

```diff
@@ -392,7 +392,15 @@ This is done as we do not except tutors to teach lessons beyond the day itself.
 </div>
 
 Example:
-* `add_session n/John Doe d/2021-01-01 t/18:00 k/120 s/Biology f/80`
```

Comment:
I think can add a <br> tag here?
![Uploading image.png…]()

Reply:
What do you mean by this?

Reply:
Psps, i meant can add a br tag between the Tip and Example?
So:
teach lessons beyond...
"<br>"
Example:

Reply:
Ah okay, have done the changes thank you!